arm-sql/2015-05-01-preview/swagger/virtualNetworkRules.json

```diff
@@ -0,0 +1,277 @@
+          },
+          "204": {
+            "description": "The specified Vnet Firewall Rule does not exist."
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/virtualNetworkRules": {
+      "get": {
+        "tags": [
+          "VnetFirewallRule"
+        ],
+        "description": "Returns information about Vnet Firewall Rules.",
+        "operationId": "VnetFirewallRules_List",
+        "x-ms-examples": {
+          "Lists the virtual network firewall rules": { "$ref": "../examples/VirtualNetworkRulesList.json" }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ServerNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved the list of vnet firewall rules.",
+            "schema": {
+              "$ref": "#/definitions/VnetFirewallRuleList"
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 404 SubscriptionDoesNotHaveServer - The requested server was not found\n\n * 404 ResourceNotFound - The requested resource was not found."
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "VnetFirewallRule": {
+      "description": "Vnet Firewall Rule Resource.",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/VnetFirewallRuleResourceProperties"
+        },
+        "location": {
+          "type": "string"
+        },
+        "id": {
+          "type": "string"
+        },
+        "name": {
+          "type": "string"
+        },
+        "type": {
+          "type": "string"
+        }
+      }
+    },
+    "VnetFirewallRuleResourceProperties": {
+      "description": "Properties of a vnet firewall rule resource.",
+      "type": "object",
+      "properties": {
+        "virtualNetworkSubnetId": {
+          "description": "The VnetSubnetId",
+          "type": "string"
+        }
+      }
+    },
+    "VnetFirewallRuleList": {
+      "description": "A list of Vnet Firewall Rule Resource.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/VnetFirewallRule"
+          }
+        },
+        "nextLink": {
```

Comment:
Do you actually have paging? If no, remove this and add x-ms-paging: { nextLinkName: null } to the list operation.